pkg/network/setup/netstat.go

```diff
@@ -159,18 +164,28 @@ func ifacesStatusFromDomainInterfaces(domainSpecIfaces []api.Interface) []v1.Vir
 			Name:       domainSpecIface.Alias.GetName(),
 			MAC:        domainSpecIface.MAC.MAC,
 			InfoSource: netvmispec.InfoSourceDomain,
+			QueueCount: domainInterfaceQueues(domainSpecIface.Driver),
 		})
 	}
 	return vmiStatusIfaces
 }
 
+func domainInterfaceQueues(driver *api.InterfaceDriver) uint {
+	if driver != nil && driver.Queues != nil {
+		return *driver.Queues
+	}
+
+	return DefaultInterfaceQueueCount
+}
+
 func sriovIfacesStatusFromDomainHostDevices(hostDevices []api.HostDevice, vmiIfacesSpecByName map[string]v1.Interface) []v1.VirtualMachineInstanceNetworkInterface {
 	var vmiStatusIfaces []v1.VirtualMachineInstanceNetworkInterface
 
 	for _, hostDevice := range filterHostDevicesByAlias(hostDevices, sriov.AliasPrefix) {
 		vmiStatusIface := v1.VirtualMachineInstanceNetworkInterface{
 			Name:       hostDevice.Alias.GetName()[len(sriov.AliasPrefix):],
 			InfoSource: netvmispec.InfoSourceDomain,
+			QueueCount: DefaultInterfaceQueueCount,
```

Comment:
Why should we report a queue count of 1 if we do not know what is the queue?
It may be that the queue is actually much higher. We can even check it on an SR-IOV setup and confirm.
(it is suppose to be HW specific)